yarn-project/end-to-end/src/prover-coordination/e2e_prover_coordination.test.ts

```diff
@@ -401,7 +401,12 @@ describe('e2e_prover_coordination', () => {
     await advanceToNextEpoch();
 
     // Wait a bit for the sequencer / node to notice a re-org
-    await sleep(2000);
+    await retryUntil(
+      async () => (await ctx.aztecNode.getTxReceipt(tx2BeforeReorg.txHash)).status === TxStatus.SUCCESS,
+      'wait for re-inclusion',
+      60,
+      1,
+    );
```

Comment:
```suggestion
    await sleep(2000);
    await retryUntil(
      async () => (await ctx.aztecNode.getTxReceipt(tx2BeforeReorg.txHash)).status === TxStatus.SUCCESS,
      'wait for re-inclusion',
      60,
      1,
    );
```

Reply:
WDYT about keeping the `sleep`? I'm worried that the retryUntil could run _too fast_ and the receipt we get is from before the reorg.